en/docs/manage-apis/design/b2b-api-management/api-consumption.md

```diff
@@ -29,39 +29,50 @@ For more information on setting up WSO2 Identity Server 7.1.0, see [Setup WSO2 I
     keytool -import -alias wso2is7cert -file is7.cert -keystore client-truststore.jks -storepass wso2carbon
     ```
 
+2. Configure the `logoutSessionStateAppender` setting to `OIDC` in each portal if API Manager and WSO2 Identity Server are running on different hosts. Skip this step if both servers are running on localhost.
 
-2. Need to add new local claim to store organization id. For that go to  Home > Identity > Claims> Add and select `Add Local Claim` and fill the form. Use Claim URI as [http://wso2.org/claims/organizationId](http://wso2.org/claims/organizationId )
+    Modify the `logoutSessionStateAppender` value to `OIDC` in these settings.json files:
+    ```
+    <AM_HOME>/repository/deployment/server/webapps/publisher/site/public/conf/settings.json
+    <AM_HOME>/repository/deployment/server/webapps/admin/site/public/conf/settings.json
+    <AM_HOME>/repository/deployment/server/webapps/devportal/site/public/theme/settings.json
+    ```
```

Comment:
Shall we compare the readability of adding these 3 file paths as a bullet list with the current code block format?